src/Microsoft.DotNet.Build.Tasks/PackageFiles/partialfacades.task.targets

```diff
@@ -61,6 +61,11 @@
       <GenFacadesBuildPartialReferenceFacade>true</GenFacadesBuildPartialReferenceFacade>
     </PropertyGroup>
 
+    <ItemGroup>
```

Comment:
Why'd you move these around?  Typically, `<FileWrites>` happen AFTER the task that writes the file.

Reply:
When the task below fails these files never get cleaned up. I hit this is error multiple times locally.

Reply:
And just to clarify, these intermediate files already exist before the task is executed because of operations above, i.e. Move